building/mac/requirements.sh

```diff
@@ -5,7 +5,7 @@ brew uninstall qt5
 brew install boost pkg-config automake
 
 # Install some deps
-brew install qt@5.5 protobuf miniupnpc openssl qrencode berkeley-db4
+brew install qt@5.10 protobuf miniupnpc openssl qrencode berkeley-db4
```

Comment:
Sorry to butt in, If the install is going to be done via homebrew the location of libraries on the `build.sh` script will need to be changed, I just installed qt@t5.10 via homebrew and the location of the libs is `/usr/local/opt/qt/`. Quoting `brew`'s post-install message.

```
This formula is keg-only, which means it was not symlinked into /usr/local,
because Qt 5 has CMake issues when linked.

If you need to have this software first in your PATH run:
  echo 'export PATH="/usr/local/opt/qt/bin:$PATH"' >> ~/.bash_profile

For compilers to find this software you may need to set:
    LDFLAGS:  -L/usr/local/opt/qt/lib
    CPPFLAGS: -I/usr/local/opt/qt/include
For pkg-config to find this software you may need to set:
    PKG_CONFIG_PATH: /usr/local/opt/qt/lib/pkgconfig

==> Summary
🍺  /usr/local/Cellar/qt/5.10.0: 9,351 files, 290MB
```